website/docs/docs/deploy/ci-jobs.md

```diff
@@ -35,6 +36,7 @@ To make CI job creation easier, many options on the **CI job** page are set to d
 
 1. Options in the **Execution settings** section:
     - **Commands** &mdash; By default, it includes the `dbt build --select state:modified+` command. This informs dbt Cloud to build only new or changed models and their downstream dependents. Importantly, state comparison can only happen when there is a deferred environment selected to compare state to. Click **Add command** to add more [commands](/docs/deploy/job-commands)  that you want to be invoked when this job runs.
+    - **Linting**<Lifecycle status="beta" /> &mdash; Enable this option for dbt to lint the SQL files in your project. If this check runs into an error, dbt can either **Fail job run** or **Continue running job**. 
```

Comment:
Need to be clear that linting runs first. That's why you might want to fail the job so that you don't have to do your PR build if you don't pass linting. This is the main value prop in having sqlfulff run natively in dbt Cloud instead of using a Github action, so I want to make sure that this run order and stuff is clear from the docs